OpenOversight/app/templates/incident_list.html

```diff
@@ -1,5 +1,18 @@
 {% extends "list.html" %}
-
+{% block title -%}
+	{%- if objects.items|length > 0 -%}
+		View all incidents for {{ objects.items[0].department.name }} - OpenOversight
+	{%- else -%}
+		No Incidents Found
+	{%- endif -%}
+{% endblock %}
+{% block meta %}
+	{% if objects.items|length > 0 %}
+		<meta name="description" content="View all incidents for {{ objects.items[0].department.name }}.">
+	{% else %}
+		<meta name="description" content="No incidents found.">
+	{% endif %}
+{% endblock %}
```

Comment:
Can we simplify both of these? And just title it "View incidents" or something regardless of if there are results or not?
I am also fine with keeping the distinction if there are no results to say "No incidents found" and use the `department` object that is already passed to the renderer and used a little further down in the file.
The `/incident` can also be called without the department id in which case all incidents in the database will be displayed not just one department